.eslintrc.js

```diff
@@ -23,7 +23,6 @@ module.exports = {
             endOfLine: "lf",
             printWidth: 80,
             tabWidth: 3,
-            trailingComma: "es5",
```

Comment:
FYI removing this causes all the extra commons to be added. I don't really care too much about which style we use here.

Reply:
This one triggers an ESLint warning with the error message on my editor I don't know what about the others.